powerline/config_files/themes/powerline_unicode7.json

```diff
@@ -143,6 +143,15 @@
 			"args": {
 				"text": "🖫⃥"
 			}
+		},
+
+		"powerline.segments.i3wm.scratchpad": {
+			"args": {
+				"icons": {
+					"fresh": "O",
+					"changed": "X"
```

Comment:
And `<C-r>=nr2char(0xXXXX)<CR>`. I usually use the latter because it allows correcting errors.